src/providers/details/tabs/Hosts/components/HostsNetworksSetUserName.tsx

```diff
@@ -0,0 +1,51 @@
+import { type Dispatch, type FC, type FormEvent, type SetStateAction, useState } from 'react';
+import { validateNoSpaces } from 'src/modules/Providers/utils/validators/common';
+
+import { FormGroupWithHelpText } from '@components/common/FormGroupWithHelpText/FormGroupWithHelpText';
+import { TextInput } from '@patternfly/react-core';
+import { useForkliftTranslation } from '@utils/i18n';
+
+import type { ValidationState } from './utils/types';
+
+type HostsNetworksSetUserNameProps = {
+  username: string;
+  setUsername: Dispatch<SetStateAction<string>>;
+};
+
+const HostsNetworksSetUserName: FC<HostsNetworksSetUserNameProps> = ({ setUsername, username }) => {
+  const { t } = useForkliftTranslation();
+
+  const [usernameValidation, setUsernameValidation] = useState<ValidationState>('default');
```

Comment:
Is ValidateState an enum? Or, please convert to an enum and use it as an initial value instead of hardcoding it